cmd/pd-server/main.go

```diff
@@ -22,6 +22,8 @@ import (
 
 	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
 	"github.com/pingcap/log"
+	dashboardAPI "github.com/pingcap/pd/pkg/dashboard/apiserver"
```

Comment:
how about `dashboardapi` and `dashboardui`, cc  [package-name](https://blog.golang.org/package-names).